src/api/uploads/BaseUpload.js

```diff
@@ -49,6 +51,7 @@ class BaseUpload extends Base {
         const attributes = {
             name: this.fileName || name,
             parent: { id: this.folderId },
+            description: this.fileDescription,
```

Comment:
Does this need to be part of the preflight OPTIONS request? If so, why?

Reply:
@danielkim328, can you comment? It looks like this field is also sent in the `preflightSuccessHandler`. Why is it sent twice?

Reply:
Ideally we want to verify the file submission is valid in preflight check, which should include whether description CAN be applied, such as valid length, or allow description etc

Reply:
Okay. I'm not sure what "ideally" means in this case, but I'm not very familiar with this endpoint, so I'll take your word for it.

Reply:
Well, preFlightRequest is made BEFORE the actual upload occurs to check whether the upload WILL be accepted. As I understand, preFlightRequest currently check only for file name conflicts, but 'ideally' it should also check if fileDescription can also be applied.